mbed_lstools/platform_database.py

```diff
@@ -478,3 +479,16 @@ def remove(self, id, permanent=False, device_type='daplink', verbose_data=False)
                     self._update_db()
 
                 return _modify_data_format(removed, verbose_data)
+
+    def update_from_web(self, url='https://os.mbed.com/api/v3/platforms/?format=json'):
```

Comment:
is this public API ? this could throw some specific exception when updating fails for some reason

Reply:
@bridadan This is a public API, and it should have a docstring, yeah?

Reply:
Yes to both! I'll make a little todo in the original post